yang/ietf-l3vpn-ntw.yang

```diff
@@ -753,38 +753,34 @@ module ietf-l3vpn-ntw {
                   description
                     "An identifier of an active VPN instance profile.";
                 }
+                leaf bearer-reference {
```

Comment:
It makes sense to make maintained under connection to group < 3 connectivity parameters.